modules/transports/core/nhttp/src/main/java/org/apache/synapse/transport/dynamicconfigurations/DynamicProfileReloader.java

```diff
@@ -0,0 +1,168 @@
+/*
+ * Copyright (c) 2005-2014, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.synapse.transport.dynamicconfigurations;
+
+import org.apache.axiom.om.OMElement;
+import org.apache.axis2.description.Parameter;
+import org.apache.axis2.description.ParameterInclude;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+
+import javax.xml.namespace.QName;
+
+/**
+ * Abstract class to use as Profile Reloader Subscribers. FileUpdateNotificationHandler will notify
+ * this once the event is triggered.
+ */
+public abstract class DynamicProfileReloader {
+
+    private static final Log LOG = LogFactory.getLog(DynamicProfileReloader.class);
```

Comment:
Use consistent names for entities such as Logger instances, etc. I've seen "log" being used in many other places. Stick to one approach in order to preserve consistency of things across the code base.